app/models/booking.rb

```diff
@@ -11,4 +16,29 @@ class Booking < ApplicationRecord
   scope :get_room_ids, ->{select(:room_id).not_delete}
 
   scope :not_delete, ->{where(deleted: 0)}
+
+  scope :get_booking_of_room, (lambda do |room_id|
+    not_delete
```

Comment:
not delete này gọi ngoài dc nhỉ, cần gì bỏ vào đây

tại controller mà gọi 
`xx.not_delete.get_booking_of_room` 


cơ mà có cần viết scope ko nhỉ, lấy room.bookings là ra toàn bộ booking của rôm rồi